deal.II-toolchain/packages/adolc.package

```diff
@@ -0,0 +1,51 @@
+# use development version?
+#CANDI_ADOLC_FROM_GIT=1
+
+if [ -n "$CANDI_ADOLC_FROM_GIT" ]; then
+    # download git repository
+    VERSION=master
+    EXTRACTSTO=ADOL-C-master
+
+    NAME=adol-c.git
+    PACKING=git
+    SOURCE=https://gitlab.com/adol-c/
+
+else
+    # download release tarball
+
+    VERSION=2.6.3
+    NAME=ADOL-C-${VERSION}
+    EXTRACTSTO=ADOL-C-${VERSION}
+    SOURCE=https://www.coin-or.org/download/source/ADOL-C/
+    CHECKSUM=f78f67f70d5874830a1ad1c0f54e54f7
+    PACKING=.tgz
+
+    VERSION=2.6.4-rc1
+    NAME=ADOL-C-${VERSION}
+    EXTRACTSTO=ADOL-C-${VERSION}
+    SOURCE=https://github.com/tjhei/adol-c/releases/download/v2.6.4-rc1/
+    CHECKSUM=cca27e9579c0c9ab4dd0c1cfa1c44433
+    PACKING=.tar.gz
+fi
+unset CANDI_ADOLC_FROM_GIT
+
+
+BUILDCHAIN=autotools
+
+BUILDDIR=${BUILD_PATH}/adolc-${VERSION}
+INSTALL_PATH=${INSTALL_PATH}/adolc-${VERSION}
+
+CONFOPTS=""
```

Comment:
if `CONFOPTS` are empty, remove it or set this line as comment (this will be set automatically reset by candi after the package is done)

Reply:
Does one use this to pass options to the configuration script? I didn't see this earlier. If this is the case then we should at the very least pass `--enable-atrig-erf`, but maybe also `--enable-advanced-branching` too. See the [spack package](https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/adol-c/package.py) for reference.

Reply:
Yes, `CONFOPTS` will be passed for `autotools` to the `configure` script. Please add the requested flags.